R/lt_id.R

```diff
@@ -76,7 +76,7 @@ lt_id_qm_a <- function(nqx, nMx, AgeInt) {
 
 #' @export
 #' @rdname lt_id_qm_a
-qxmx2ax <- lt_id_qm_a
+#'qxmx2ax <- lt_id_qm_a
```

Comment:
i'll have to investigate `lt_id_qm_a`. What made you want to comment out that line?